game_functions.py

```diff
@@ -1,30 +1,32 @@
 #technically battle functions but oh whale
+#eventually revamp cobat system including defend and eventually add weapons and armor.
 from random import randint
 import game_commands
 import monster_module
 import game_parser
 
-def is_player_dead(player):
-    if(player["hit_points"] <= 0):
+def is_someone_dead(character):
+    if(character["hit_points"] <= 0):
         return True
     else:
         return False
 
 # def game_over_screen():
 
-
-def is_someone_dead(player, monster):
-    if(monster["hit_points"] <= 0):
-        print(f"\n{player['name']} has slain {monster['name']}!")
+def are_two_someones_dead(character_1, character_2):
```

Comment:
This isn't a particularly good function. Somewhere, where you know WHY you care if any entity is dead, you should call a generic `is_dead(someone)` function and do the right thing there. This is kind of pushing context "down" when you should endeavor, always, to push it "up".